packages/itwinui-css/src/avatar/avatar.scss

```
@@ -48,7 +48,9 @@
     font-size: var(--_iui-avatar-font-size);
   }
 
-  > .iui-icon {
+  > .iui-icon,
+  > svg,
+  > .iui-svg-icon {
```

Now that I think about it, `iui-icon` is actually unnecessary since we can just target `svg` without `>`.
```suggestion
  svg {
```